app.rb

```diff
@@ -10,49 +10,90 @@
 
 # Use enumerables to -
 
+#warm-up
+#get the first apartment
+# puts apartment[0]
```

Comment:
There is no `apartment` variable/method, only a `apartments` so this will error out